doc/dependency_decisions.yml

```diff
@@ -1,65 +1,65 @@
 ---
-- - :whitelist
```

Comment:
Be sure you rebase master, this file and `doc/licenses/licenses.xml` have changed [here](https://github.com/3scale/porta/commit/751c22c4a35e0bf96d7d10c85ac156eec7b66075)